salt/states/monit.py

```diff
@@ -74,22 +70,20 @@ def unmonitor(name):
     try:
         for key, value in result.items():
             if "Not monitored" in value[name]:
-                ret["comment"] = ("{0} is not being monitored.").format(name)
+                ret["comment"] = ("{} is not being monitored.").format(name)
```

Comment:
```suggestion
                ret["comment"] = "{} is not being monitored.".format(name)
```
And other occurrences of this pontless use of `()`, please.